public/scripts/timeline.js

```diff
@@ -1597,6 +1608,21 @@ function isValidDate(dateString)
     return day > 0 && day <= monthLength[month];
 };
 
+function generateMementoURIList(object)
+{
+    var URIM = [];
+
+    $.each(object,function(index,obj){
+        if($(obj.event_html).attr("src").indexOf("notcaptured") < 0){
+            URIM.push(obj.event_link);
+        }
+    });
+    var download = document.getElementById("downloadMementoURI");
+    download.href = "";

```

Comment:
Is this initialization/resetting of `href` attribute necessary here?

Reply:
No, that line is not necessary. It has been removed.